internal/public.proto

```diff
@@ -40,7 +40,6 @@ message QueryRequest {
 	string Query = 1;
 	repeated uint64 Slices = 2;
 	bool ColumnAttrs = 3;
-	string Quantum = 4;
 	bool Remote = 5;
```

Comment:
that's correct. 
if a client library is using `QueryRequest`, the proto messages need to match. changing it to `bool Remote = 4` would break that.